projects/vdk-control-cli/src/vdk/internal/control/command_groups/info_group/info.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+import click
+import click_spinner
+from vdk.internal.control.configuration.defaults_config import load_default_team_name
+from vdk.internal.control.exception.vdk_exception import VDKException
+from vdk.internal.control.rest_lib.factory import ApiClientFactory
+from vdk.internal.control.utils import cli_utils
+from vdk.internal.control.utils.output_printer import OutputFormat
+
+log = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Get Control Service Information. "
+    "Get the current version of the Control Service and the supported python versions."
+)
+@click.option(
```

Comment:
Why do we need to pass team? If I understand correctly, the output of the command would not be linked to a specific job or team, it will apply to the whole Control Service deployment.